src/source/feeds.rst

```diff
@@ -31,6 +78,635 @@ XML file.
    Any change in the ad (including it re-appearing in the feed if it wasn't present the previous time) will update the
    ad and trigger image re-processing.
 
+======= ====================================================
+Example	<ad:vendorId>15839942</ad:vendorId>
+======= ====================================================
+
+.. index:: externalId
+.. _feed_externalId:
+
+externalId
+""""""""""
+.. warning::
+   There is still an **externalId** field in the XSD, this field is replaced by **vendorId**.
+   Please update your XML to reflect this change. This makes naming consistent between feeds and sellside API.
+   The **vendorId** field in the feeds has the same meaning and constraints as the **vendorId** field in the
+   sellside API.
+
+.. index:: sellerName
+.. _feed_sellerName:
+
+sellerName
+""""""""""
+
+Display name of the seller (max. 60 characters).
+
+======= ====================================================
+Example	<ad:sellerName>Cups, Caps &amp; Craps</ad:sellerName>
+======= ====================================================
+
+.. index:: title
+.. _feed_t:
+
+title
+"""""
+
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1024 characters. See :ref:`categories`. URLs are not allowed as part of the title.
+
+======= ====================================================
+Example	<ad:title>Goedkope A-merk herenfietsen</ad:title>
+======= ====================================================
+
+.. index:: description
+.. _feed_descr:
+
+description
+"""""""""""
+
+The description field of the ad.
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1MB characters. See :ref:`categories`. URLs are not allowed as part of the description.
+All HTML elements except for the ones below will be removed:
+
+.. code-block:: html
+
+    <u> <em> <ul> <li> <p> <strong> <br> <blink>*
+    
+*\*paid extra*
+
+======= =========================================================================================================
+Example .. code-block:: html 
+    
+            <ad:description><![CDATA[
```

Comment:
do you want to explicitly mention that we recommend this description block to be encapsulated in !CDATA blocks?

Reply:
Good idea